src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -516,16 +526,13 @@ public String getInvalidityInfo(FieldType fieldType, String fieldName, Object va
     public String getValidityInfoForSizeCappedAlphanumericNonEmptyString(String fieldName, int maxLength, String value) {
         
         Assumption.assertTrue("Non-null value expected for "+fieldName, value != null);
-        String sanitizedValue = Sanitizer.sanitizeForHtml(value);
         
-        if (value.isEmpty()) {
-            return String.format(SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE, value, fieldName, REASON_EMPTY, fieldName, maxLength);
+        if (value.isEmpty() || value.length()>maxLength) {
```

Comment:
Conditional check is combined because I combined the error message for empty string and string too long.
(As suggested in the original issue thread at [this comment][link].)

[link]: https://github.com/TEAMMATES/teammates/issues/4129#issuecomment-133441150